containertools/cmd/nbwatch/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+	"path/filepath"
+
+	"k8s.io/klog/v2"
+
+	"github.com/fsnotify/fsnotify"
+)
+
+func main() {
+	if err := run(); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func run() error {
+	w, err := fsnotify.NewWatcher()
+	if err != nil {
+		return err
+	}
+	defer w.Close()
+
+	w.Add("/content/src")
+
+	go watchLoop(w)
```

Comment:
why does `watchLoop` need to be started in a goroutine? I see it has an endless for loop in it anyway and then you could get rid of the block forever line?

Reply:
Thats a good catch, I ripped this from the fsnotify example. I think in our case, we could avoid the goroutine.